### PR TITLE
feat(favorites): #MED-288 add the differentiation between manuals and external_resources

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

add field is_textbook=true for textbooks to differentiate them from external_resources
add icon for favorite cards

image

## Checklist tests

check json result for different request
check the display of icon (allow to differentiate manuals from external resources)

## Issue ticket number and link

[MED-220](https://jira.support-ent.fr/browse/MED-220)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: backend/src/main/java/fr/openent/mediacentre/source/GAR.java
Comment (niko-amosse) [inline backend/src/main/java/fr/openent/mediacentre/source/GAR.java:1]: ici on check si il y a MAN dans typePresentation, si c'est le cas c'est un texbook et donc on ajout un champ is_textbook

**Comment by Mattox0** [comment]
Tu as pu tester que ça marchait bien ?

**Comment by Mattox0** [comment]
On veut aussi ajouter l'icone pour les cartes de recherches sur toutes les pages

**Comment by Mattox0** [comment]
tu dois aussi changer le model dans le back

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**